dev_tools/scripts/scw

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+if `command -v sem`.empty?
+  abort <<~ERROR
+    sem CLI not found
+
+    brew install semaphoreci/tap/sem
+
+    docs: https://docs.semaphoreci.com/reference/sem-command-line-tool/
+  ERROR
+end
+
+branch = `git branch --show-current`.strip
+
+workflows = `sem get workflows`.split("\n")
```

Comment:
Might be able to do 
```suggestion
workflows = `sem get workflows`.lines
```
But this may not be as transferrable across all ruby versions we have.